src/monitoring_service/constants.py

```diff
@@ -1,8 +1,11 @@
 import textwrap
 from datetime import timedelta
 
-DEFAULT_REQUIRED_CONFIRMATIONS: int = 10
-MAX_FILTER_INTERVAL: int = 100_000
+from raiden.utils.typing import BlockTimeout
+
+DEFAULT_FILTER_INTERVAL: BlockTimeout = BlockTimeout(1_000)
+MAX_FILTER_INTERVAL: BlockTimeout = BlockTimeout(100_000)
+MIN_FILTER_INTERVAL: BlockTimeout = BlockTimeout(2)
```

Comment:
Nothing to do with your PR but that type alias name is pretty confusing...

Reply:
Definitively, I forget the exact name every time.